packages/wallet/src/transfer.test.ts

```diff
@@ -19,6 +19,34 @@ describe('Wallet', () => {
     expect(receiverBalances).toEqual([{ assetId: NativeAssetId, amount: BigNumber.from(1) }]);
   });
 
+  it('can transfer with custom TX Params', async () => {
+    const provider = new Provider('http://127.0.0.1:4000/graphql');
+
+    const sender = await generateTestWallet(provider, [[100, NativeAssetId]]);
+    const receiver = await generateTestWallet(provider);
+
+    /* Error out because gas is to low */
+    try {
+      await sender.transfer(receiver.address, 1, NativeAssetId, {
+        gasLimit: 1,
+        gasPrice: 1,
+        bytePrice: 1,
+        maturity: 1,
+      });
+    } catch (e) {
+      const error = e as Error;
+      expect(error.message).toContain('InsufficientGas');
+    }
```

Comment:
tried to do `expect().toThrow()` but didnt work.  Is there another way?

Reply:
expect(async () => {
      await sender.transfer(receiver.address, 1, NativeAssetId, {
        gasLimit: 1,
        gasPrice: 1,
        bytePrice: 1,
        maturity: 1,
      });
}).toThrow....

Reply:
Figured it out! Didn't realize you have to `await` the expect

```
   await expect(async () => {
      await sender.transfer(receiver.address, 1, NativeAssetId, {
        gasLimit: 1,
        gasPrice: 1,
        bytePrice: 1,
        maturity: 1,
      });
    }).rejects.toThrowError('InsufficientGast');
```